addons/portal_rating/static/src/js/portal_chatter.js

```diff
@@ -139,6 +157,24 @@ PortalChatter.include({
         return params;
     },
 
+    /**
+     * re-render rating card with new values
+     * @private
+     */
+    _renderRatingCard: function () {
+        var self = this;
+        this._rpc({
```

Comment:
yes, we need a new route to get the rating stats for updating rating statistics.  messageFetch update only messages with rating value(not rating statistics). rating statistics update the only when the chatter widget initializes (using _chatterInit) into the dom.
here we have split the code related to the rating statistics see '_updateRatingCardValues' method which is called from _chatterInit and _renderRatingCard to update rating statistics card layout.

or

we can update the 'portal_message_fetch' route same as like (related to the rating stats part) 'portal_chatter_init' route for updating rating stats??

Reply:
> we can update the 'portal_message_fetch' route same as like (related to the rating stats part) 'portal_chatter_init' route for updating rating stats??

It would probably be better, yes.